acceptance/bundle/deploy/secret-scope/script

```diff
@@ -0,0 +1,15 @@
+envsubst < databricks.yml.tmpl > databricks.yml
+
+cleanup() {
+    trace $CLI bundle destroy --auto-approve
+}
+trap cleanup EXIT
+
+trace $CLI bundle deploy
+trace $CLI bundle summary --output json | jq '.resources.secret_scopes.secret_scope1'
+
+trace $CLI secrets list-scopes -o json | jq '.[] | select(.name == "my-secrets")'
+trace $CLI secrets list-acls my-secrets
+
+trace $CLI secrets put-secret my-secrets my-key --string-value "my-secret-value"
+trace $CLI secrets get-secret my-secrets my-key
```

Comment:
While useful in general, it is out of scope for this test.

Reply:
i'd like to test here that a deployed secret scope can actually be used to read/write secrets. would you suggest removing these commands completely?